### ISSUE TITLE
Error when running attempting to scrape with rvest

### ISSUE THREAD

**Issue by sjp** [sjp/selectr#9]
(Reported via email)

After upgrading from version 0.3-2 to 0.4-0, the following code no longer works:

```r
library(R.utils)
library(tidyverse)
library(rvest)
"https://edition.cnn.com/" %>% read_html %>% html_node(".edition-picker__current-edition")
```

We expect to see the following result:

```rconsole
{xml_node}
<div class="edition-picker__current-edition">
[1] <span class="nav-section__expand-icon">+</span>
```

But instead get:

```rconsole
Error in assign(name, value, envir = attr(static, ".env")) : 
  use of NULL environment is defunct
```

**Comment by sjp** [comment]
After some investigation, this is caused by the use of the `R.utils` package. The core bit of code that `selectr` is attempting to execute can be reproduced to a simpler set of steps as `rvest` is not relevant for reproducing the problem.

When we use a clean environment and run either one of the following lines, everything works correctly:

```rconsole
> selectr::css_to_xpath(".asd") # simple case
[1] "descendant-or-self::*[@class and contains(concat(' ', normalize-space(@class), ' '), ' asd ')]"
> selectr::css_to_xpath(".edition-picker__current-edition") # reported example selector
[1] "descendant-or-self::*[@class and contains(concat(' ', normalize-space(@class), ' '), ' .edition-picker__current-edition ')]"
```

However, with the `R.oo` package attached (performed when `R.utils` is attached from the reported scenario) the code no longer runs.

```rconsole
> library(R.oo)
> selectr::css_to_xpath(".asd")
Error in assign(name, value, envir = attr(static, ".env")) : 
  use of NULL environment is defunct
```

The reason for this occurring is that the `R6` class `Class` (not exported by the `selectr` package) clashes with the `R.oo` base class object which is also named `Class`. A simple fix is to rename the `R6` class in `selectr` to `ClassSelector`. It's a bit of a hack but it should solve the problem and avoid issues with `R.oo`.

**Comment by sjp** [comment]
This should now be fixed in 48e0a3b05c87b54323b2b4fbbc71b6e6f37f566c.

I'll close this issue once it is confirmed fixed via email correspondence.

**Comment by sjp** [comment]
Confirmed fixed.
